dcos_launch/README.md

```diff
@@ -1,21 +1,23 @@
 # `dcos-launch` User Guide
 ## Overview
 
-`dcos-launch` is a portable linux executable to create, wait for provisioning, describe, test/validate, and destroy an arbitrary cluster. `dcos-launch` can be used as an installed library for development, but it is also shipped as an executable with every DC/OS build. E.g. to get the current `testing/master` version of DC/OS, use this url:
+`dcos-launch` is a portable linux executable to create, wait for provisioning, describe, test/validate, and destroy an arbitrary cluster.
+
+`dcos-launch` can be used as development tool, but it is also shipped as an executable with every DC/OS build. E.g. to get the current `testing/master` version of DC/OS, use this url:
```

Comment:
we no longer ship this tightly coupled to dcos but the link is still correct.  I will make another edit after this